.github/workflows/website_build_dispatch.yml

```diff
@@ -0,0 +1,16 @@
+name: Dispatch trigger for deepchem website re-build
```

Comment:
Add a section to the docs in the CI part that mentions this dispatch trigger and documents it

Reply:
https://deepchem.readthedocs.io/en/latest/development_guide/ci.html, https://github.com/deepchem/deepchem/blob/master/docs/source/development_guide/ci.rst